packages/prosess-aarskvantum-oms/src/ÅrskvantumIndex.tsx

```diff
@@ -75,12 +78,35 @@ const ÅrskvantumIndex = ({
     submitCallback,
     behandling,
     saerligSmittevernAp: apForVurderÅrskvantumDok,
-    aktiviteter: sisteUttaksplan?.aktiviteter
+    aktiviteter: sisteUttaksplan?.aktiviteter,
   };
 
+  const featureTogglesData = restApiHooks.useGlobalStateRestApiData<{ key: string; value: string }[]>(
+    K9sakApiKeys.FEATURE_TOGGLE,
+  );
+  const featureToggles: FeatureToggles = useMemo<FeatureToggles>(
```

Comment:
Kan denne trekkes ut som egen funksjon? Ser den finnes to steder fra før av

Reply:
Trakk den ut i en hook